packit_service/models.py

```diff
@@ -104,6 +104,7 @@ class JobTriggerModelType(str, enum.Enum):
     branch_push = "branch_push"
     release = "release"
     issue = "issue"
+    merge_request = "merge_request"
```

Comment:
As we discussed, I would avoid creating new trigger names. GitLab users should be familiar with the term pull-request.

(We can think about aliasing later -- please, create an issue so we don't forget if you think it's important.)